src/models/local/LocalBaseController.js

```diff
@@ -45,6 +45,7 @@ export default class LocalBaseController {
       log.error(`Contract ${contractName} has an explicit constructor. Move it to an initializer function to use it with ZeppelinOS.`)
     }
 
+    if (!this.packageData.contracts) this.packageData.contracts = {};
```

Comment:
We shouldn't be initializing the contracts list here, that's part of the `init` command. Have you initialized your project before running `add`?

Reply:
I have. I added more details in #230.